conf/openmetadata.yaml

```diff
@@ -142,6 +143,12 @@ authenticationConfiguration:
   clientId: ${AUTHENTICATION_CLIENT_ID:-""}
   callbackUrl: ${AUTHENTICATION_CALLBACK_URL:-""}
 
+jwtTokenConfiguration:
```

Comment:
@harshach - Is there a configuration missing to provide location (path) for the below ?

Reply:
users are expected to configure absolute paths and this is optional configuration.